### ISSUE TITLE
DataTransferObject.h file not found // Lexical or Preprocessor Issue

### ISSUE THREAD

**Issue by martnst** [rollbar/rollbar-apple#45]
**Describe the bug**
The latest version does not build. 

**To Reproduce**
1. New XCode Project iOS App with Storyboard 
2. Add with SwiftPM by URL `https://github.com/rollbar/rollbar-apple.git`& verision 1.12.9
3. CMD + B to Build

**Screenshots**
![image](https://user-images.githubusercontent.com/3084607/108573047-ff50f280-7313-11eb-8133-10f76ec4a38d.png)

**Rollbar-Apple SDK version: v1.12.9**

**Calling the SDK from (x-mark all suitable):**
[] Objective-C
[x] Swift
[] Other: 

**Runtime environment (please complete the following information as applicable):**
 - Device: MacBook Pro (15-inch, 2016)
 - OS: macOS Big Sur 11.1 (20C69)
 - XCode: Version 12.4 (12D4e)

**Comment by akornich** [comment]
@martnst , are you sure you are using Rollbar-Apple SDK? This SDK was never released as v1.12.9 and only had so far releases matching the following versioning pattern: v2.0.0-alphaNN.
I assume you are integrating the SDK via SwiftPM. You can look into the following SDK demo app: https://github.com/rollbar/rollbar-apple/tree/master/Demos/macosAppSwiftSPM for details.
You can register the SDK SPM package like so:
![Screen Shot 2021-02-19 at 23 57 07](https://user-images.githubusercontent.com/1753993/108589432-0b888000-7313-11eb-8a93-35bb07d1c098.png)

**Comment by martnst** [comment]
Hi @akornich, 

thanks for your fast response. Yes, I did install it via SwiftPM. I just update my post to be more clear. 

I can confirm that it is now working when linking to the master branch. Version 1.12.9 was suggested to use by XCode. I simply followed the https://docs.rollbar.com/docs/apple#via-swift-package-manager guide. From there I went with the latest version XCode came up with. 

Bests, 
Martin
